composer.json

```diff
@@ -51,5 +56,10 @@
         "branch-alias": {
             "dev-master": "1.1.x-dev"
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "symfony/flex": true
+        }
```

Comment:
Why was this needed? It looks like we already allow in CI

Reply:
good catch